vendor/vendor.json

```diff
@@ -647,6 +647,12 @@
 			"versionExact": "v1.12.19"
 		},
 		{
+			"checksumSHA1": "3AcW4E/TxmNXfklpo1szXmY+QzE=",
+			"path": "github.com/aws/aws-sdk-go/service/glue",
+			"revision": "3e9e64d88a3aba2c745efa24d45620400c527f04",
+			"revisionTime": "2017-11-03T19:05:25Z"
```

Comment:
Could you write the command you used to include these vendors?
It is missing the SDK version, which is an important information in our way to manage vendors!

Reply:
oops, my bad. Have upped a new version using `govendor fetch github.com/aws/aws-sdk-go/service/glue@v1.12.22`